packages/path_provider/path_provider_windows/lib/src/path_provider_windows_real.dart

```diff
@@ -208,7 +207,7 @@ class PathProviderWindows extends PathProviderPlatform {
       // From that, load the VERSIONINFO resource
       final int infoSize = GetFileVersionInfoSize(moduleNameBuffer, unused);
       if (infoSize != 0) {
-        infoBuffer = calloc<Uint8>(infoSize);
+        infoBuffer = calloc<BYTE>(infoSize);
         if (GetFileVersionInfo(moduleNameBuffer, 0, infoSize, infoBuffer) ==
             0) {
           calloc.free(infoBuffer);
```

Comment:
Do we still need the manual frees? I was hoping finalize support would let us simplify all this code, but I haven't followed the details of adoption in FFI.

Reply:
Yeah, w\We do for now. The reason for the `package:ffi` version bump is to allow us to use a different memory allocation API for Windows, which should in principle us to use `NativeFinalizer` moving forward. This is the prerequisite to that.